tests/backend_test.py

```diff
@@ -1528,15 +1528,15 @@ def test_optimisation_level_3_compilation() -> None:
     compiled_3 = b.get_compiled_circuit(c, 3)
     compiled_3_timeout = b.get_compiled_circuit(c, 3, timeout=0)
 
-    assert compiled_2.n_2qb_gates() == 68
-    assert compiled_2.n_gates == 186
-    assert compiled_2.depth() == 130
+    assert compiled_2.n_2qb_gates() <= 69
```

Comment:
This is the one that most concerns me. Previously this circuit compiled to 68 2-qubit gates at optimization level 2; now it compiles to 69. The circuit contains no conditional gates so it's hard to see where this could be coming from...

Reply:
I've experimented with changing the parameters of the circuit (numbers of iterations of the two loops), and it seems that performance is sometimes worse, sometimes the same, sometimes better. I think we can probably put this down to different behaviour of `lightsabre` in qiskit 2, so happy to accept these numbers.